src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -473,18 +487,29 @@ export const CheckoutPayment = ({
           <Description>
             {textKeys.CHECKOUT_PAYMENT_DETAILS_DESCRIPTION()}
           </Description>
-          <div id="dropin-container" ref={adyenRef}></div>
+          {isPaymentConnected ? (
+            <>
+              <PaymentResult>
+                <ThinTick color={colorsV3.gray900} />
+                {textKeys.CHECKOUT_PAYMENT_ADYEN_SETUP_DONE_MESSAGE()}
+              </PaymentResult>
+              <Button
+                fullWidth={true}
```

Comment:
on what line did you see the isLoading? I removed it from the button, only left it on the footer.

Reply:
Sorry, just assumed that the button would have a loading state since it was passed into the footer.

Reply:
ah, I didn't understand your question initially. That is also a great comment, will add the previous loading booleans as part of the disabled prop so that if data is still loading the button will be disabled.